build.sbt

```diff
@@ -38,7 +39,6 @@ lazy val root = project.in(file("."))
       Libraries.fs2,
       Libraries.fs2Io,
       Libraries.doobieCore,
-      Libraries.doobiePostgres,
```

Comment:
This can be removed as it is no [longer in use](https://github.com/snowplow-incubator/igluctl/pull/123/commits/b6da3061128d6aaa07adf6cf02a546459778f601)